crates/runtime-ffi/src/autostart.rs

```diff
@@ -11,15 +13,30 @@ impl AutostartConfigManagerFfi {
         Ok(Self(AutostartConfigManager::new(data_root_path.into())?))
     }
 
-    pub fn enable(&self) -> RuntimeResultFfi<()> {
-        Ok(self.0.enable()?)
+    pub fn enable(&self, config: RuntimeNetworkConfigFfi) -> RuntimeResultFfi<()> {
```

Comment:
The auto-start manager now stores the runtime network config, which should be applied when auto-starting the runtime.